### DISCUSSION TITLE
Driver issues on Windows 11 Guest

### DISCUSSION THREAD

**Discussion by datarepos76** [utmapp/UTM#5926] [UTM Help (macOS)]
Hi everybody,
I recently started having the following problem on my MacBooks under UTM. When creating new Windows machines, the device drivers are apparently no longer installed correctly. The picture in the device manager is absolutely identical before and after the installation of the Guest Tools. The screen resolution can also no longer be adjusted correctly.  The problem occurs identically on both MacBooks, see below. I can no longer pinpoint exactly when the problem started, but it seems to have come with a macOS or UTM update. Do any of you have similar problems? I did some screenshots, see attachments.
Kind regards
Mathias

![drivers 2](https://github.com/utmapp/UTM/assets/145758494/69ef9afd-1fb0-4e56-93df-2b84ff5880cb)
![drivers 3](https://github.com/utmapp/UTM/assets/145758494/357d8f1b-04ff-4058-8af4-2530b10bd8e2)
![drivers 1](https://github.com/utmapp/UTM/assets/145758494/2dd45ca8-46f7-45ba-8d6a-74328760bc3c)

**Comment by datarepos76** [comment]
Here are my system details.

MacBook Air M2 on Sonoma 14.1.1, UTM 4.4.4, Windows 11 23H2 and 22H2, Guest Tools 0.229
MacBook Pro M1 Pro on Sonoma 14.1.1, UTM 4.4.4, Windows 11 23H2 and 22H2, Guest Tools 0.229

**Comment by ibousfiha** [comment]
I'm having the same issue using M3 Max
Any updates on this ?

**Comment by G5Petr** [comment]
Hi I have the same issue on MacBookPro M2 running Ventura 13.4 and UTM 4.4.5 with Win11 insider 26016 and Guest Tools 0.229. Not sure if it is due to this issue, but I am not able to connect USB configured devices with it's own (only Win) configuration apps (i.e. Bose PowerMatch amplifier). USB drives are working fine (Mass Storage shown in devman), tried both USB2/USB3 in UTM settings. In VM it is connected but Device manager shows it in "Other devices" with correct name. I wanted to find the driver but I have just *.dll already installed and driver wizard wants me to find some *.inf file. Any idea please?

**Comment by datarepos76** [comment]
Hi UTM Team,
It's great that you are converting my issues into discussions. But does anyone care about solving the problem? Unfortunately, there has been no response from your team so far. What a pity.
All the best
datarepos

**Comment by juanjgacosta** [comment]
Hi I have a similar issue on MacBookPro M2 running Sonoma 14.6,1 and UTM 4.3.5 with Windows 11 Insider ARM64 25905 and guest tools 0.164.4. I can't connect a specific USB device even after installing the corresponding driver.

**Reply by juanjgacosta** [reply]
I finally managed to solve the problem using this windows program in the virtual machine: https://zadig.akeo.ie/#google_vignette

**Comment by G5Petr** [comment]
Hi, I have no progress with the issue. I just hope for other connection possibilities on installed products - direct Mac USB (Mac app), web browser, IP based app (could be Win). Currently I need Windows just to run some only Win apps to configure audio/video devices. So I have to use my more than 10 years old MBPro with VirtualBox, where USB worked correctly for both flash drives and connected devices.

**Comment by datarepos76** [comment]
I have still not received any response from the UTM team. Unfortunately, they just don't seem to be interested in this topic.

**Comment by Rainerxyz** [comment]
Same problem here. This is really annoying. I also tried Windows 10 in emulation-mode and found out that drivers are installed properly but there is a different problem, still preventing me from carrying out what my aim is. I'm using a serial to usb converter, so the driver is creating a com-port but when the software is trying to access this port, it complains the port being occupied by another process. But guess what, there is no other program running that would access the port. On my old mac running Virtual box with Win10, all was working fine. Unfortunately Win10 and Win11 are not (yet) supported in Vbox on an Mac silicon. So not sure if I can do what I need to do. Anyone who can solve this?

**Comment by ModelSD** [comment]
Running into the same frustrating problem. The device drivers are installed correctly but the Windows device manager doesn't seem to actually search for them. It seems to skip to "not found" immediately.

**Comment by jzarzeckis** [comment]
I'm experiencing the same issue too with Windows 11 Arm
Mac Sequoia 15.5
UTM Version 4.6.5 (108)

**Comment by wickedyoda** [comment]
Having the same issue, tried the above resolution and others. Nothing has worked.

**Comment by justa-randomuser** [comment]
for me imma put windows 11 drivers on a usb then installing them

**Comment by patrickmetz** [comment]
Use [the official UTM guide for Windows 11](https://docs.getutm.app/guides/windows/) and fetch the Apple Silicon Image of Windows 11 via Crystal fetch. That just works™.
